packages/common/src/util.ts

```diff
@@ -191,9 +195,13 @@ exports.setErrorStack = (error, newStack) => {
  * @param {Object} obj - object to update
  * @returns {Object} a shallow clone of the object with updated property name
  *
+ * @deprecated
+ *
  * @alias module:util
  */
-exports.renameProperty = (from, to, obj) => {
+// @ts-ignore
+export const renameProperty = (from, to, obj) => {
```

Comment:
Why are you using `@ts-ignore` on some of the deprecated functions and not on others?

Reply:
It was purely a "this looks hard to migrate and we're gonna get rid of it anyway ..." decision.